File: Ejercicios Propuestos 3/Ejercicio 2/Ejercicio 2.py
NAME = "Renzo Leonardo Cienfuegos Cardenas"
GROUP = 2

# Solicitar la cantidad de notas a registrar
cantidad = int(input("Ingrese la cantidad de notas a registrar: "))

# Crear una lista vacía para almacenar las notas
vector_notas = []

# Inicializar la variable que acumulará la suma
suma = 0.0

# Solicitar las notas una por una
print("Ingrese las notas una por una:")

# Recorrer desde la posición 0 hasta cantidad - 1
for i in range(cantidad):
    # Leer la nota
    nota = float(input(f"Ingrese la nota para la posición {i}: "))
    
    # Guardar la nota en el vector
    vector_notas.append(nota)
    
    # Acumular la suma
    suma += nota

# Calcular el promedio
promedio = suma / cantidad

# Mostrar el resultado
print("El promedio de las notas es:", promedio)

File: Ejercicios Propuestos 3/Ejercicio 2/ejercicio2.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cout << "Ingrese la cantidad de notas: ";
    cin >> n;

    float nota[100];
    float suma = 0;
    for (int i = 0; i < n; i++) {
        cout << "Nota " << i << ": ";
        cin >> nota[i];
        suma = suma + nota[i];
    }

    float promedio = suma / n;
    cout << "El promedio es: " << promedio << endl;
    return 0;
}
